webapp/app/components/sortable-header/component.js

```diff
@@ -0,0 +1,14 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+    downArrow: '&#9660',
```

Comment:
You don't need to put these as variables. Put them directly inside the template.

Also, consider font awesome icons
https://fontawesome.com/icons?d=gallery&q=down